src/main/java/com/joejoe2/chat/service/channel/GroupChannelService.java

```diff
@@ -64,10 +66,55 @@ GroupMessageDto inviteToChannel(String fromUserId, String toUserId, String chann
   GroupMessageDto acceptInvitationOfChannel(String ofUserId, String channelId)
       throws UserDoesNotExist, ChannelDoesNotExist, InvalidOperation;
 
-  GroupMessageDto removeFromChannel(String fromUserId, String targetUserId, String channelId)
+  /**
+   * let administrator remove target user from the channel
+   *
+   * @param adminId user id of the administrator
+   * @param targetUserId id of target user
+   * @param channelId id of target channel
+   * @throws UserDoesNotExist
+   * @throws ChannelDoesNotExist
+   * @throws InvalidOperation
+   */
+  GroupMessageDto removeFromChannel(String adminId, String targetUserId, String channelId)
       throws UserDoesNotExist, ChannelDoesNotExist, InvalidOperation;
 
-  GroupMessageDto leaveChannel(String ofUserId, String channelId)
+  GroupMessageDto leaveChannel(String userId, String channelId)
```

Comment:
All new methods in this interface require Javadoc I think.

Reply:
ok